app/controllers/api/v2/application_controller.rb

```diff
@@ -38,19 +30,9 @@ def check_terms_accepted
         render json: error_body, status: :forbidden
       end
 
-      def email_from_token(token)
-        if Settings.authentication.algorithm == 'plain-text'
-          # This method can be used in development mode to simplify querying
-          # the API with curl. It should allow us to do:
-          #
-          #    curl -H 'Authorization: Bearer user@education.gov.uk' http://localhost:3001/api/v2/providers
-          token
-        else
-          (payload, _options) = JWT.decode(token,
-                                               Settings.authentication.secret,
-                                               Settings.authentication.algorithm)
-          payload['email']
-        end
+      def assign_sentry_contexts
+        Raven.user_context(id:              @current_user&.id)
+        Raven.tags_context(sign_in_user_id: @current_user&.sign_in_user_id)
```

Comment:
Would it make sense to use `Raven.user_context` here, assuming it works? Then maybe it'd show up in the `user` section at the end of the tags:

![image](https://user-images.githubusercontent.com/15608/56674637-4a673380-66b2-11e9-898b-c7835e60f259.png)

Reply:
It doesn't accept custom values as far as I can tell, whereas tags are a key/value store.

Reply:
😿